mantidproject/mantid
# Correct the image of SampleShapePlot on the MeshPlotHelp docs page

**Description of work.**
Update SampleShapePlot image on this page https://docs.mantidproject.org/nightly/plotting/MeshPlotHelp.html to one with the correct axes (see diff).

**To test:**

Not exactly necessary but you could build docs-html and check the new image is used.

*There is no associated issue.*


*This does not require release notes* because **minor docs update**

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
